src/cljs/rems/table.cljs

```diff
@@ -161,27 +161,41 @@
 (defn- filterable? [column]
   (:filterable? column true))
 
-(defn- display-row? [row filtered-columns filters]
-  (if (empty? filtered-columns)
-    true ; table has no filtering enabled
-    (some (fn [column]
-            (str/includes? (str (get-in row [(:key column) :filter-value]))
-                           filters))
-          filtered-columns)))
+(defn- display-row? [row filtered-columns search-terms]
+  (or (empty? filtered-columns) ; table has no filtering enabled
+      (empty? search-terms)
+      (every? (fn [search-term]
+                (some (fn [column]
+                        (str/includes? (str (get-in row [(:key column) :filter-value]))
+                                       search-term))
+                      filtered-columns))
+              search-terms)))
+
+(defn parse-search-terms [s]
+  (->> (re-seq #"\S+" (str s))
+       (map str/lower-case)))
+
+(deftest test-parse-search-terms
```

Comment:
does doo find this?

Reply:
Yes. There are others in the same file too.